README.md

```diff
@@ -1 +1,55 @@
-# perl-Mojo-WebSocketProxy
\ No newline at end of file
+# perl-Mojo-WebSocketProxy
+[![Build Status](https://travis-ci.org/binary-com/perl-Mojo-WebSocketProxy.svg?branch=master)](https://travis-ci.org/binary-com/perl-Mojo-WebSocketProxy) 
+
+Using this module you can forward websocket JSON-RPC 2.0 requests to RPC server.
```

Comment:
can you beef up the README more? explain at a higher level about how it can be used